src/client/commands/submit.rs

```diff
@@ -149,7 +167,7 @@ pub async fn submit_computation(
         .collect();
     args.insert(0, opts.command.into());
 
-    let submit_cwd = opts.cwd.unwrap_or_else(|| std::env::current_dir().unwrap());
+    let submit_cwd = opts.cwd;
```

Comment:
This should be resolve "." to absolute path. Or is there still a discussion what should be default cwd?

Reply:
I think that it should be resolved in the worker, not the submitter, but this is just a temporary placeholder so that tests pass. Let's discuss the defaults after we properly implement placeholder replacing during submit.

Reply:
Ok, but then lets change at least the default stdout to %{SUBMIT_DIR}/stdout....

Reply:
There's no point in doing that until we replace SUBMIT_DIR :) Actually, I will implement the replacement in this PR, there's no reason to divide the PRs.